### ISSUE TITLE
Very minor user guide update for Alt keys with iterm2 on Mac

### ISSUE THREAD

**Issue by khs26** [zellij-org/zellij#3630]
The user guide links a helpful site for passing alt + <arrow keys> through to zellij when using iterm2 on Mac. However, I think [this comment](https://www.reddit.com/r/zellij/comments/13twru4/comment/kpmsjv2/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) has a more complete solution and might make sense to reference.

**Comment by khs26** [comment]
To quote the comment (for posterity):

> iTerm2 Settings -> [Profile] -> Keys -> Left Option Key, press the 'Esc+' radio button. Upon doing that, Alt + n or any combination of Alt keys for zellij should work on iterm2

**Comment by imsnif** [comment]
Thanks! I'll take a look. Hope all is good with you otherwise.

**Comment by khs26** [comment]
Yes, it is thanks! We have a one-year-old and a very slightly younger puppy, since we're gluttons for punishment. Free time is something we're hoping might happen in a year or so. ;) Hope all is well with you too - the project seems to be going very nicely!

**Comment by WellingtonFonseca** [comment]
ProductName:		macOS
ProductVersion:		15.0
BuildVersion:		24A335

I had the same problem and I did this step by step to solve it

1. Open iTerm2.
1. Go to the Preferences menu (⌘ + ,).
1. Select the Profiles tab and then choose the profile you are using.
1. Go to the Keys section.
1. There, you will see options for "Left Option Key" and "Right Option Key".
1. Change the behavior of these keys to "Esc+". This will make Alt send escape sequences instead of typing special characters like ~.
